styles/designs/carnival/parts/_exercises-components.scss

```diff
@@ -33,8 +34,63 @@ $InjectedExercise__Question__Wrapper: (
     )
 );
 
-$Exercise__Problem__Container: exerciseChildContainer((child: problem, name: "ProblemContainer", hasImageTableOnTop: false));
-$Exercise__Problem__Container--WithImageTableOnTop: exerciseChildContainer((child: problem, name: "ProblemContainerWithImageTableOnTop", hasImageTableOnTop: true));
```

Comment:
In function `exerciseChildContainer` selectors are not direct children. Since `os-problem-container` appears twice in exercises with multipart questions and has different properties depends on what is its parent, we have to be more specific in selectors. For now, I created new components in this file instead of changing function, because then we would have to test books from all themes.